lndr-backend/src/Lndr/Handler/Credit.hs

```diff
@@ -159,6 +158,23 @@ createPendingRecord pool signedRecord = do
     void . liftIO . withResource pool $ Db.insertPending signedRecord
 
 
+calculateSettlementCreditRecord :: ServerConfig -> CreditRecord -> CreditRecord
+calculateSettlementCreditRecord _ cr@(CreditRecord _ _ _ _ _ _ _ _ _ _ Nothing _) = cr
+calculateSettlementCreditRecord config cr@(CreditRecord _ _ amount _ _ _ _ _ ucac _ (Just currency) _) =
```

Comment:
This may be idiomatic Haskell, and if so, that's fine. When I see this, I think of the game of Hangman, which I feel is a pattern akin to Magic Numbers (except in this case I suppose it's "magic underscores"). It **is** clear enough to read, given the clear variable names here, so my commentary is merely a 🚲🏚remark.